modules/applications/pages/ui-builder/how-to-handle-queries-dependency.adoc

```diff
@@ -0,0 +1,82 @@
+= How to handle queries dependency
+:page-aliases: applications:how-to-handle-queries-dependency.adoc
```

Comment:
:page-aliases: is useless when the page is not an existing page

```suggestion
```

Reply:
I didn't add it initially, but it is required to handle the path (in pages/ui-builder), as it is done for other guides.